auto_backup/models/db_backup.py

```diff
@@ -83,6 +83,16 @@ class DbBackup(models.Model):
              "read permissions for that file.",
     )
 
+    backup_format = fields.Selection(
+        [
+            ("zip", "zip (includes filestore)"),
+            ("dump", "pg_dump custom format (without filestore)")
+        ],
+        string="Backup Format",
```

Comment:
No need to write `string="Backup Format",`

Reply:
Question of style. I prefer explicit declaration that implicit declaration.
Has OCA guide on this that I didn't see?

Reply:
See https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#fields

Reply:
👍